cmd/lotus-provider/proving.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli/v2"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/dline"
+
+	"github.com/filecoin-project/lotus/provider"
+)
+
+var provingCmd = &cli.Command{
+	Name:  "proving",
+	Usage: "Utility functions for proving sectors",
+	Subcommands: []*cli.Command{
+		//provingInfoCmd,
+		provingCompute,
+	},
+}
+
+var provingCompute = &cli.Command{
+	Name:  "compute",
+	Usage: "Compute a proof-of-spacetime for a sector (requires the sector to be pre-sealed)",
+	Subcommands: []*cli.Command{
+		provingComputeWindowPoStCmd,
+	},
+}
+
+var provingComputeWindowPoStCmd = &cli.Command{
```

Comment:
Might be nice to have a way to submit a mock-compute task to the database, so the computation isn't done in the local cli, but on the real workers (which imo is what SPs will expect in most cases - running check tasks on the cluster with proper scheduling, but no submitting things to the chain)

Reply:
that makes sense. Would you want it to do that with live updates, or just direct them to logs and database to evaluate the success?

Reply:
We could have either this or a separate command poll for success